ethereum/dex/liquidity/insert_balancer_v2.sql

```diff
@@ -0,0 +1,158 @@
+CREATE OR REPLACE FUNCTION dex.insert_liquidity_balancer_v2(start_ts timestamptz, end_ts timestamptz=now()) RETURNS integer
+LANGUAGE plpgsql AS $function$
+DECLARE r integer;
+BEGIN
+WITH days as ( -- update table entries until previous day
+    SELECT day FROM generate_series(start_ts, (SELECT end_ts - interval '1 day'), '1 day') g(day)
+),
+balancer_v2_pools as ( -- https://github.com/duneanalytics/abstractions/blob/master/labels/ethereum/balancer_v2_pools.sql
+    SELECT
+        pool_id,
+        SUBSTRING(pool_id FOR 20) as pool_address,
+        token_address,
+        normalized_weight,
+        symbol as pool_name_symbolic
+    FROM
+    (
+    select c."poolId" as pool_id, unnest(cc.tokens) as token_address, unnest(cc.weights)/1e18 as normalized_weight, cc.symbol, 'WP' as pool_type
+    from balancer_v2."Vault_evt_PoolRegistered" c
+    inner join balancer_v2."WeightedPoolFactory_call_create" cc
+    on c.evt_tx_hash = cc.call_tx_hash
+    union all
+    select c."poolId" as pool_id, unnest(cc.tokens) as token_address, unnest(cc.weights)/1e18 as normalized_weight, cc.symbol, 'WP2T' as pool_type
+    from balancer_v2."Vault_evt_PoolRegistered" c
+    inner join balancer_v2."WeightedPool2TokensFactory_call_create" cc
+    on c.evt_tx_hash = cc.call_tx_hash
+    union all
+    select c."poolId" as pool_id, unnest(cc.tokens) as token_address, 0 as normalized_weight, cc.symbol, 'SP' as pool_type
+    from balancer_v2."Vault_evt_PoolRegistered" c
+    inner join balancer_v2."StablePoolFactory_call_create" cc
+    on c.evt_tx_hash = cc.call_tx_hash
```

Comment:
Could you remove these lines? Stable pools don't have weights so null is better suited. Since you only use this CTE to get the token weights, you can delete this and the stable pools' weights will be null by default.
```suggestion
```

Reply:
Done. Added in latest commit and merged to production.